db/config.cc

```diff
@@ -714,6 +714,7 @@ db::config::config(std::shared_ptr<db::extensions> exts)
     , enable_dangerous_direct_import_of_cassandra_counters(this, "enable_dangerous_direct_import_of_cassandra_counters", value_status::Used, false, "Only turn this option on if you want to import tables from Cassandra containing counters, and you are SURE that no counters in that table were created in a version earlier than Cassandra 2.1."
         " It is not enough to have ever since upgraded to newer versions of Cassandra. If you EVER used a version earlier than 2.1 in the cluster where these SSTables come from, DO NOT TURN ON THIS OPTION! You will corrupt your data. You have been warned.")
     , enable_shard_aware_drivers(this, "enable_shard_aware_drivers", value_status::Used, true, "Enable native transport drivers to use connection-per-shard for better performance")
+    , cpu_sharding_algorithm_name(this, "cpu_sharding_algorithm_name", value_status::Used, "biased-token-round-robin", "Select the type of connection-per-shard algorithm")
```

Comment:
Need to explain the options, and to validate them. If the user selects xyz, everything will break.

Reply:
I'll try look on other ones that are verified as part of config.cc